File: 07143023/43_163_Return_Day/python/return_day.py
def return_day(number: int) -> str or None:
    """Takes in a number 1-7 and returns the corresponding day of the week with day 1 starting with Sunday."""
    days: dict = {1: "Sunday", 2: "Monday",
            3: "Tuesday", 4: "Wednesday",
            5: "Thursday", 6: "Friday",
            7: "Saturday"}
    
    if number < 8 or number > 0:
        print(days.get(number))


return_day(1) # "Sunday"
return_day(2) # "Monday"
return_day(3) # "Tuesday"
return_day(4) # "Wednesday"
return_day(5) # "Thursday"
return_day(6) # "Friday"
return_day(7) # "Saturday"
return_day(41) # None

File: 07143023/43_163_Return_Day/java/ReturnDay.java
import java.util.HashMap;

public class ReturnDay {
    public static void main(String[] args) {
        System.out.println(returnDay(1));
        System.out.println(returnDay(2));
        System.out.println(returnDay(3));
        System.out.println(returnDay(4));
        System.out.println(returnDay(5));
        System.out.println(returnDay(6));
        System.out.println(returnDay(7));
        System.out.println(returnDay(41));
    }

    public static String returnDay(int number) {
        HashMap<Integer, String> days = new HashMap<>();

        days.put(1, "Sunday");
        days.put(2, "Monday");
        days.put(3, "Tuesday");
        days.put(4, "Wednesday");
        days.put(5, "Thursday");
        days.put(6, "Friday");
        days.put(7, "Saturday");

        if (number <= 7 && number >= 1) {
            String answer = days.get(number);
            return answer;
        } else {
            return "Please pick a number between 1 and 7.";
        }
    }
}
